calc_bc.html

```diff
@@ -594,6 +594,13 @@
                             <input class="btn-input" type="checkbox" id="helpingHandR" />
                             <label class="btn btn-xxwide" for="helpingHandR">Helping Hand</label>
                         </div>
+                            <div class="left" title="Has this Pokemon's power been boosted by Defense Curl?">
+                                <input class="btn-input" type="checkbox" id="defenseCurlL" />
+                                <label class="btn btn-xxwide" for="defenseCurlL">Defense Curl</label>
+                        </div>
+                        <div class="right" title="Has this Pokemon's power been boosted by Defense Curl?">
+                            <input class="btn-input" type="checkbox" id="defenseCurlR" />
+                            <label class="btn btn-xxwide" for="defenseCurlR">Defense Curl</label>
```

Comment:
The alignment and tag matching is still wrong here, should look like this: http://hastebin.com/hepasibixe.html